api/build.gradle

```diff
@@ -243,7 +249,10 @@ dependencies {
 
 swaggerSources {
```

Comment:
Here or as a follow-up change, it might also make things clearer if we moved backend APIs into a subdir, e.g. /backends or /firecloud.

Reply:
Let's do that as a follow-up.